app/config/locale/templates/email-base-cloud.tpl

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+    <head>
+        <link
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=DM+Sans:wght@500;600&display=swap"
+            rel="stylesheet"
+        />
+        <style>
+            .main a {
+                color: currentColor;
+            }
+            .main {
+                padding: 32px;
+                line-height: 1.5;
+                color: #616b7c;
+                font-size: 15px;
+                font-weight: 400;
+                font-family: "Inter", sans-serif;
+            }
+            table {
+                width: 100%;
+                border-spacing: 0;
+            }
+            table,
+            tr,
+            th,
+            td {
+                margin: 0;
+                padding: 0;
+            }
+            td {
+                vertical-align: top;
+            }
+            h1 {
+                font-size: 22px;
+                margin-bottom: 0px;
+                margin-top: 0px;
+                color: #373b4d;
+            }
+            h2 {
+                font-size: 20px;
+                font-weight: 600;
+                color: #373b4d;
+            }
+            h3,
+            td h3 {
+                font-size: 14px;
+                font-weight: 500;
+                color: #373b4d;
+                line-height: 21px;
+                margin: 0;
+                padding: 0;
+            }
+            h4 {
+                font-family: "DM Sans", sans-serif;
+                font-weight: 600;
+                font-size: 12px;
+                color: #4f5769;
+                margin: 0;
+                padding: 0;
+            }
+            hr {
+                border: none;
+                border-top: 1px solid #e8e9f0;
```

Comment:
If we set colors, we must set background color of body to white. Otherwise some mail clients with dark theme support would make it look bad. Example:
![CleanShot 2024-01-12 at 10 30 39@2x](https://github.com/appwrite/appwrite/assets/19310830/9c967cb6-cafd-4c6c-9e1c-446f8e04f04a)

![CleanShot 2024-01-12 at 10 30 58@2x](https://github.com/appwrite/appwrite/assets/19310830/14b37950-9b93-48c4-8c42-235a4aaf5c47)

Reply:
I agree, I prefer it to look good rather than have 1/2 support in dark mode.

Reply:
Forcing the background color to white for now until we make a decision